DEVELOPER_GUIDE.md

```diff
@@ -43,8 +43,8 @@ One easy way to get Java 11 on *nix is to use [sdkman](https://sdkman.io/).
 ```bash
 curl -s "https://get.sdkman.io" | bash
 source ~/.sdkman/bin/sdkman-init.sh
-sdk install java 11.0.2-open
-sdk use java 11.0.2-open
+sdk install java 21.0.2-open
+sdk use java 21.0.2-open
```

Comment:
I understand the java version should also be updated, but it is not related to the current PR. Can you raise new PR?

Reply:
sure, no problem. #908